docs/manual/external-tools.tex

```diff
@@ -499,6 +499,14 @@
    the result is to use the Checker
    Framework compiler instead of the regular \<javac>.)
 
+   \item
+   Option 4:
```

Comment:
Isn't this option missing putting `checker.jar` on the classpath? This option will use the right javac, but it won't have the Checker Framework classes on the classpath and the examples in the manual assume that.

Reply:
This confuses me too.  It's per instructions in https://docs.google.com/document/d/1mcOJUrYzHDMp2MfPztOmOaaSFjOWZquUN1fO_LCmXuo/edit# .  I'm just going to delete it.